src/ContentHubClientCommonTrait.php

```diff
@@ -13,6 +13,13 @@
    */
   trait ContentHubClientCommonTrait {
 
+    /**
+     * The last call's response object.
+     *
+     * @var \Psr\Http\Message\ResponseInterface
+     */
+    private ResponseInterface $response;
```

Comment:
In this version of the library we still support 7.3. That would break backward compatibility.